test/ngResource/resourceSpec.js

```diff
@@ -338,6 +338,17 @@ describe("resource", function() {
   });
 
 
+  it('should encode & in query params unless in query param value', function() {
```

Comment:
I'm wondering if part of this test is a duplicate of the one preceding this test.